docs/_static/vcr/v2.yaml

```diff
@@ -210,6 +210,38 @@ paths:
                 $ref: '#/components/schemas/VCVerificationResult'
         default:
           $ref: '../common/error_response.yaml'
+  /internal/vcr/v2/verifier/vp:
+    post:
+      summary: Verifies a Verifiable Presentation
+      description: |
+        Verifies a Verifiable Presentation. It checks:
+        * Signature of the verifiable presentation and the verifiable credentials
+        * Expiration
+        * Revocation status
+        * If the issuers of the verifiable credentials are trusted
+        * If the issuers of the verifiable credentials were not deactivated at time of issuing
+
+        error returns:
+        * 400 - One or more of the given parameters are invalid
```

Comment:
I would expect one of the above rules to fail since the call has no params.

Reply:
I'll change the 400 description to parameters or VP.

Also, 400 is returned when the VP itself can't be verified (e,g. invalid signature, not trusted), which is ambiguous. So i'll change that to 422